dev-tools/install.sh

```diff
@@ -72,6 +72,7 @@ cd $(dirname "$BASH_SOURCE")/..
 if [ "$1" == "--clean" ]; then
   echo "Removing installed dependencies and compiled static files."
   rm -rf .venv node_modules src/cms/static
+  python3 -m venv .venv
 fi
 
```

Comment:
Maybe something along the lines of:
```suggestion
if [ "$1" == "--pipenv-in-venv" ] && [ ! -d ".venv" ]; then
    python3.7 -m venv .venv
    . .venv/bin/activate
    pip3 install pipenv
fi
```